ansible/includes/ansible.yml

```diff
@@ -7,6 +7,7 @@
 - name: Add ansible PPA repository
   apt_repository:
     repo: "ppa:ansible/ansible"
+  when: ansible_distribution_version < '20.04'
```

Comment:
Should this also check that the distro is Ubuntu to guard against us going mad in the future and something some other distro?

Reply:
Yeah, agreed, as above I guess depends how you want to approach it. I'm happy to re-work it to make it all a bit more distro aware, but for now I was going with the "only Ubuntu is supported" mantra. Let me know which way you wanna go and I'll update it see what you think.

Reply:
Yeah, I think I addressed this above. Thanks for your patch by the way!